src/plugins/opensearch_dashboards_usage_collection/server/collectors/management/schema.ts

```diff
@@ -47,12 +47,12 @@ export const stackManagementSchema: MakeSchemaFrom<UsageStats> = {
   'visualization:tileMap:WMSdefaults': { type: 'text' },
   'timeline:target_buckets': { type: 'long' },
   'timeline:max_buckets': { type: 'long' },
```

Comment:
Are we not changing these back to `timelion` ?

Reply:
I need to check about this like I'm not sure where these values are going or if it's strictly for usage collection. Like while running it should be thinking it's `timeline` so I thought usage-wise it we help to know it's the application `timeline` but if it's required for saved objects.